src/integration/java/org/springframework/data/couchbase/repository/N1qlCouchbaseRepositoryTests.java

```diff
@@ -125,4 +129,131 @@ public void shouldPageThroughSortedEntities() {
       previousAttendees = party.getAttendees();
     }
   }
+
+  @Test
+  public void shouldFindAllByExample() {
+    Party party = new Party();
+    party.setName("party");
+    Example<Party> sample = Example.of(party, ExampleMatcher.matching()
+            .withStringMatcher(ExampleMatcher.StringMatcher.CONTAINING).withIgnoreNullValues().withIgnorePaths("attendees"));
+    Iterable<Party> parties = repository.findAll(sample);
+    int count = 0;
+    for(Party p:parties) {
+      count +=1;
+    }
+    assertTrue(count == 14);
```

Comment:
in general, try to avoid `assertTrue` and `assertFalse` other than on boolean fields, the more specific assertions will usually display the unexpected value, etc...